### PR TITLE
Multi vm network values

### PR DESCRIPTION
**Pull request by steversmith**
This PR fixes issue: https://github.com/chef/chef-provisioning-vagrant/issues/33 

Modify the handling of the vm.network setting to detect if the value is an array:
- If so, then generate a config.vm.network(xxx) setting for each value in the array.
- If not, then generate a single config.vm.network(xxx) setting as before.

The following vagrant_options:

```
options = {
  vagrant_options: {
    'vm.box' => 'redhat',
    'vm.network' => [
      ':private_network, {ip: "192.168.33.34"}',
      ':forwarded_port, guest: 27017, host: 27117'
    ]
  }
}

machine 'somemachine' do
  machine_options options
end
```

Produces the following vagrant file, which has both settings in a form compatible with vagrant:

```
Vagrant.configure('2') do |outer_config|
  outer_config.vm.define "somemachine" do |config|
    config.vm.box = "redhat"
    config.vm.network(:private_network, {ip: "192.168.33.34"})
    config.vm.network(:forwarded_port, guest: 27017, host: 27117)
    config.vm.hostname = "somemachine"
  end
end
```

### CODE REVIEW

**Comment by steversmith** [comment]
@SolverDude Here it is

File: lib/chef/provisioning/vagrant_driver/driver.rb [hunk 1]
```diff
@@ -222,7 +223,13 @@ def create_vm_file(action_handler, vm_name, vm_file_path, machine_options)
           end
           merged_vagrant_options.each_pair do |key, value|
             if key == 'vm.network'
-              vm_file_content << "    config.#{key}(#{value})\n"
+              if value.is_a? Array
```
Comment (tyler-ball) [inline lib/chef/provisioning/vagrant_driver/driver.rb]: To DRY this up some you can do

```ruby
vm_networks = [value].flatten
vm_networks.each do |network|
  vm_file_content << "    config.#{key}(#{network})\n"
end
```

**Comment by tyler-ball** [comment]
1 comment, otherwise it is ready for merge

**Comment by steversmith** [comment]
@tyler-ball Nice! Greate to learn a new ruby technique (at least new to me).

**Comment by SolverDude** [comment]
I'm guessing 'Feedback Requested' is for anyone consuming this.  I think it's clean and a good fix for what was lacking.

**Comment by tyler-ball** [comment]
@SolverDude The `Feedback Requested` has two purposes.  Typically it is asking for feedback/changes from the PR author (or opinions from the public).  It is also a marker for us to look at the issue first when doing internal triage

**Comment by adam-carbone** [comment]
Did this make it into 'chef-provisioning-vagrant (0.10.0)' ?

**Comment by tyler-ball** [comment]
This was just released as part of 0.11.0!

**Comment by SolverDude** [comment]
Woot!!

**Comment by steversmith** [comment]
Probably should close issue https://github.com/chef/chef-provisioning-vagrant/issues/33

**Pull request merged**